assets/shared/structure/structure-store.js

```diff
@@ -28,149 +29,207 @@ export function registerStructureStore( {
 	storeName,
 	getEndpoint,
 	saveError,
+	fetchError,
 	clearError,
 	updateBlock,
 	readBlock,
+	...store
 } ) {
 	const DEFAULT_STATE = {
 		serverStructure: null,
+		editorStructure: null,
 		isSavingStructure: false,
-		hasStructureUpdate: false,
+		hasUnsavedServerUpdates: false,
+		hasUnsavedEditorChanges: false,
+		clientId: null,
 	};
 
 	const actions = {
 		/**
 		 * Fetch structure data from REST API.
 		 */
 		*loadStructure() {
-			const endpoint = yield* getEndpoint();
-			const result = yield apiFetch( {
-				path: `/sensei-internal/v1/${ endpoint }?context=edit`,
```

Comment:
The drop of `context=edit` for the course structure endpoint is making it so draft lessons no longer appear in the course outline editor.

Reply:
Added this back to both course and quiz in 44f84ec.